tests/test_templatetags.py

```diff
@@ -2392,3 +2392,215 @@ def test_inner_slot_iteration_nested_with_slot_default_and_outer_scope_variable_
             {'inner': ['ITER2_OBJ1', 'ITER2_OBJ2']} default
             """,
         )
+
+
+class ScopedSlotTest(BaseTestCase):
+    def test_slot_data(self):
+        @component.register("test")
+        class TestComponent(component.Component):
+            template: types.django_html = """
+                {% load component_tags %}
+                <div>
+                    {% slot "my_slot" abc=abc 123=var123 %}Default text{% endslot %}
```

Comment:
Using 123 as an attribute here is super strange, as neither HTML or python allows them. But I guess OK that it still works? :)

Reply:
Changed the key name in https://github.com/EmilStenstrom/django-components/pull/495/commits/40c2b1af9b6878a922cd2456b76dbd69ba1e0fef